armi/physics/executers.py

```diff
@@ -195,13 +195,14 @@ def run(self):
         # must either write input to CWD for analysis and then copy to runDir
         # or not list it in inputs (for optimization)
         self.writeInput()
-        with directoryChangers.ForcedCreationDirectoryChanger(
-            self.options.runDir,
+        with directoryChangers.TemporaryDirectoryChanger(
+            root=self.options.runDir,
             filesToMove=inputs,
             filesToRetrieve=outputs,
             outputPath=outputDir,
         ) as dc:
             self.options.workingDir = dc.initial
+            self.options.runDir = dc.destination
```

Comment:
@opotowsky sorry I marked this ready for review before it was working. I had to add this line to get it to work, but this feels sort of strange to me. What are your thoughts?

Reply:
That's interesting this is needed for the temp changer but not the forced. Wouldn't you need to change the runDir back after this?

Reply:
oh goodness I'm being dense, ignore that first sentence. this is a random dir created with the runDir as root. 

But still, would changing the runDir in this step have any downstream impacts for a case run?

Reply:
> But still, would changing the runDir in this step have any downstream impacts for a case run?

No, changing the `runDir` shouldn't have any effects. We just need the `DirectoryChanger` and the `Executer` to agree about where the executable is going to be run.

Reply:
@mgjarrett I just wanted to clear up the above question. I'm sure this is fine but I'm not visualizing where this happens in the run.